### ISSUE TITLE
2 view train on 8 GPU

### ISSUE THREAD

**Issue by jgq111** [HorizonRobotics/Uni3R#5]
Thank you for your wonderful work!
I want to know why the loss doesn't decrease during 2-view training on 8 RTX 4090 GPUs using the script you provided?

**Comment by Xiangyu1Sun** [comment]
The model looks work fine when we try to train it and can converge. Could you provide more details like training log?

**Comment by Xiangyu1Sun** [comment]
How many epochs here? In our case, the image_loss converge lower to 0.1 at 10 epochs. For language loss weight, you could try different number. And please make sure you are using the newest update code.

**Comment by jgq111** [comment]
I’m using the latest version you released, and the environment remains the same as before without any changes.

**Comment by jgq111** [comment]
Would it be possible for you to share the training loss logs from the 2-view setup? Also, may I confirm if the learning rate used was 0.0001?

**Comment by Xiangyu1Sun** [comment]
<img width="253" height="143" alt="Image" src="https://github.com/user-attachments/assets/c162cf7f-a855-4052-b9a4-5722da5847e1" />this is the setting we are using.

**Comment by jgq111** [comment]
<img width="1202" height="339" alt="Image" src="https://github.com/user-attachments/assets/41fdbc08-c482-45c4-9074-62f2303a99f6" />I was wondering if this library is https://github.com/nerfstudio-project/gsplat, because the installation guide you provided seems incomplete. Also, is this log corresponding to a 4-GPU training?

**Comment by Xiangyu1Sun** [comment]
ahh, sorry for the confusion. You still need to install gsplat and pytorch3d to train the whole model. And actually we are not using diff-gaussian-rasterization in current code version.

**Comment by jgq111** [comment]
scannet and scannet++?

**Comment by jgq111** [comment]
<img width="2449" height="1028" alt="Image" src="https://github.com/user-attachments/assets/ef83f753-272c-4b39-a658-d054cd07c403" />Because the loss increases again later and fluctuates around 0.4.

**Comment by Xiangyu1Sun** [comment]
sure, but how to check that. I download the 2 views code and it works fine on our server.
